python/ray/_private/ray_constants.py

```diff
@@ -222,6 +222,12 @@ def env_set_by_user(key):
     "%(asctime)s\t%(levelname)s ({component}) %(filename)s:%(lineno)s -- %(message)s"
 )
 
+# A dict_keys object containing the standard attributes of a LogRecord. This is used to
+# help us determine which attributes constitute user-provided context.
+LOGRECORD_STANDARD_ATTRS = logging.makeLogRecord({"message": "test"}).__dict__.keys()
```

Comment:
This can be put directly into `formatters.py` since it's not a public constant

Reply:
Updated https://github.com/ray-project/ray/pull/45098/commits/85289e6c24cff9a0ebd9e879c59cf2312c8813c6